README.md

```diff
@@ -57,6 +57,30 @@ somewhere in your CSS
 ...
 ```
 
+## Multisorting
```

Comment:
I'd go with "Sorting by multiple columns". I think "multisort" is not totally clear.